test-requirements.txt

```diff
@@ -1,11 +1,7 @@
-six
-pyOpenSSL
-flake8
-flake8-mutable
-flake8-print
-pylint
-setuptools-lint
+# General Deps
```

Comment:
```
$ head -1 requirements.txt 
ansible>=2.2
```

The current `requirements.txt` allows us to test any version greater than or equal the minimum required version.

There is no problem with specifying more specific versions in `tox.ini`.